components/centraldashboard/frontend/main-page.html

```diff
@@ -170,13 +195,20 @@
                     {text: 'Compare run output', icon: 'donut-large'},
                     {text: 'Share solution on AI Hub', icon: 'donut-large'},
                 ]},
+                quickLinks: {type: Array, value: [
+                    {text: 'Open docs', link: 'https://www.kubeflow.org/docs/started/getting-started/'},
+                    {text: 'Open Github', link: 'https://github.com/kubeflow/kubeflow'},
+                ]},
                 sidebarItemIndex: {type: Number, value: 0},
                 primaryViewIndex: {type: Number, value: 0},
                 homeOrIframeViewIndex: {type: Number, value: 0},
                 url: {type: String, value: ""},
+                buildVersion: {type: String, value: "1.0"},
```

Comment:
Can this value be retrieved programmatically? If not, the latest Kubeflow release was `0.4.1`

Reply:
This will be, and for now I will statically add 0.4.1

This will be dynamic once the backend is rewritten.

Projected features: https://docs.google.com/document/d/1-kxDUbtd16OnlTuKsHOBO0bDQ3uJzrmt6vQZk2msddc/edit?ts=5c62d226#